WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -969,11 +966,30 @@ private static int GetInputTarget(int utxoCount, int minUtxoCountTarget, WasabiR
 		return targetInputCount;
 	}
 
-	private async Task<IEnumerable<TxOut>> ProceedWithOutputRegistrationPhaseAsync(uint256 roundId, ImmutableArray<AliceClient> registeredAliceClients, CancellationToken cancellationToken)
+	private async Task<(bool, IEnumerable<TxOut>)> ProceedWithOutputRegistrationPhaseAsync(uint256 roundId, ImmutableArray<AliceClient> registeredAliceClients, CancellationToken cancellationToken)
 	{
 		// Waiting for OutputRegistration phase, all the Alices confirmed their connections, so the list of the inputs will be complete.
 		var roundState = await RoundStatusUpdater.CreateRoundAwaiter(roundId, Phase.OutputRegistration, cancellationToken).ConfigureAwait(false);
 		var roundParameters = roundState.CoinjoinState.Parameters;
+
```

Comment:
I would not put the code here instead have a dedicated method for this check and call that from StartRoundAsync.

Reply:
This can have its own methods i think, sure. I'll see if it's possible to call it from  `StartRoundAsync`

Reply:
done